client/blocks/comments/block-editor/autocompleters/user.jsx

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable wpcalypso/jsx-classname-namespace */
 
-export default suggestions => {
+export default ( suggestions ) => ( {
```

Comment:
How did this breakage happen? It looks quite serious. Was it the codemod?

Reply:
Yes, the codemod changed it for some reason so I reverted it back.